Doc/library/os.rst

```diff
@@ -1037,17 +1037,27 @@ as internal buffering of data.
    .. availability:: Unix.
 
 
-.. function:: fstat(fd)
+.. function:: fstat(fd, *, fast=False)
 
    Get the status of the file descriptor *fd*. Return a :class:`stat_result`
    object.
 
-   As of Python 3.3, this is equivalent to ``os.stat(fd)``.
+   As of Python 3.3, this is equivalent to ``os.stat(fd, fast=fast)``.
```

Comment:
```suggestion
   As of Python 3.3, this is equivalent to ``os.stat(fd, fast=False)``.
```

Reply:
😆 whoops

Reply:
Wait, no, this is correct. There's a `fast` argument to this function, which gets passed through (it just doesn't happen to make any difference right now, but that's why `fast` doesn't _guarantee_ it'll leave out any information. Sometimes we have to take the slow path regardless, and that's the case here.)

Reply:
oh, right, I missed that  :)